File: src/main/java/com/msg/gauth/domain/auth/presentation/dto/response/RefreshResponseDto.java
package com.msg.gauth.domain.auth.presentation.dto.response;

import com.fasterxml.jackson.annotation.JsonFormat;
import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;

import java.time.ZonedDateTime;

@AllArgsConstructor
public class RefreshResponseDto {
    @JsonProperty
    String accessToken;
    @JsonProperty
    String refreshToken;
    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
    ZonedDateTime expiresAt;
}

File: src/main/kotlin/com/msg/gauth/domain/auth/presentation/dto/response/RefreshResponseDto.kt
package com.msg.gauth.domain.auth.presentation.dto.response

import com.fasterxml.jackson.annotation.JsonFormat
import com.fasterxml.jackson.annotation.JsonProperty
import java.time.ZonedDateTime

data class RefreshResponseDto(
    @JsonProperty
    val accessToken: String,

    @JsonProperty
    val refreshToken: String,

    @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ss")
    val expiresAt: ZonedDateTime
)